README.md

```diff
@@ -29,7 +29,16 @@ Navigate to `localhost:8080/components/my-element/demo.html`
   * `-b <browsername>` use this browser instead of default (ex: 'Google Chrome Canary')
   * `-H <hostname>` use this hostname instead of localhost
 
-## Run Tests
+## Contributing
+
+```
+git clone git@github.com:polymerlabs/polyserve.git
+cd polyserve/
+npm install
+npm run build
```

Comment:
We include this section in the polymer CLI readme:

```

## Compiling from Source

    $ npm run build

You can compile and run the CLI from source by cloning the repo from Github and then running `npm run build`. But make sure you have already run `npm install` before building.
```

Lets try to use the same copy in both READMEs. Would this have been enough info for you, as a first time TS-er?